core/src/main/java/org/axonframework/eventsourcing/eventstore/DomainEventStream.java

```diff
@@ -139,6 +140,22 @@ static DomainEventStream concat(DomainEventStream a, DomainEventStream b) {
         return new ConcatenatingDomainEventStream(a, b);
     }
 
+    /**
+     * Filters a DomainEventStream. In the resulting stream events from stream
+     * {@code a} will be filtered by {@ filter}.
+     *
+     * @param a
```

Comment:
Nit: It seems indenting is a little off here. Shouldn't the `@param` and it's description live on the same line? Having them on the same line resembles the other comment blocks in this repository a little better.